charts/velero/values.yaml

```diff
@@ -86,6 +86,9 @@ lifecycle: {}
 # Pod priority class name to use for the Velero deployment. Optional.
 priorityClassName: ""
 
+# The number of seconds to allow for graceful termination of the pod. Optional.
+terminationGracePeriodSeconds: 3600
```

Comment:
Since it's an optional field and the Kubernetes `terminationGracePeriodSeconds` default value is 30 seconds. I think 3600 seconds changes the default time.

Reply:
True, it does, but shouldn't the default of the Helm chart be suited to the use case? Under common conditions I think Velero can easily take over thirty seconds to create a backup.

Or are you suggesting that this needs to be a minor version bump as opposed to a patch version bump?

Reply:
> True, it does, but shouldn't the default of the Helm chart be suited to the use case? Under common conditions I think Velero can easily take over thirty seconds to create a backup.

Agree with you.